### PR TITLE
add plugin settings category

### PR DESCRIPTION
**Pull request by 12944qwerty**
Adds the plugin category

Untested as I just coded this on my chromebook

### CODE REVIEW

**Review by asportnoy — CHANGES_REQUESTED** [review]
- General Settings, Plugins, Themes, and Quick CSS are all showing in the plugins settings section
- Make the section header an i18n key and name it "Plugin Settings" instead of just "Plugins"
- Sort by the plugin name instead of ID; some of them are visually out-of-order since the ID is different from the name.

File: src/Powercord/coremods/settings/index.js [hunk 1]
```diff
@@ -48,11 +48,11 @@ async function patchSettingsComponent () {
...
     if (changelog) {
       const coreSections = Object.keys(powercord.api.settings.tabs)
-        .filter(s => ['pc-commands', 'pc-i18n', 'pc-moduleManager', 'pc-notices', 'pc-settings', 'pc-updater'].indexOf(s) >= 0)
+        .filter(s => ['pc-moduleManager-themes', 'pc-moduleManager-css', 'pc-moduleManager-plugins', 'pc-general', 'pc-updater'].indexOf(s) >= 0)
         .map(s => _makeSection(s));
       const pluginSections = Object.keys(powercord.api.settings.tabs)
-        .filter(s => ['pc-commands', 'pc-i18n', 'pc-moduleManager', 'pc-notices', 'pc-settings', 'pc-updater'].indexOf(s) < 0)
-        .sort()
+        .filter(s => ['pc-moduleManager-themes', 'pc-moduleManager-css', 'pc-moduleManager-plugins', 'pc-general', 'pc-updater'].indexOf(s) < 0)
+        .sort((a, b) => powercord.api.settings.tabs[a].label < powercord.api.settings.tabs[b].label)
```
Comment (lleyton) [inline src/Powercord/coremods/settings/index.js:55]: let's use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) instead

**Review by lleyton — CHANGES_REQUESTED** [review]

File: src/Powercord/coremods/settings/index.js [hunk 2]
```diff
@@ -47,14 +47,27 @@ async function patchSettingsComponent () {
 
     const changelog = sections.find(c => c.section === 'changelog');
     if (changelog) {
-      const settingsSections = Object.keys(powercord.api.settings.tabs).map(s => _makeSection(s));
+      const coreSections = Object.keys(powercord.api.settings.tabs)
+        .filter(s => ['pc-moduleManager-themes', 'pc-moduleManager-css', 'pc-moduleManager-plugins', 'pc-general', 'pc-updater'].indexOf(s) >= 0)
```
Comment (asportnoy) [inline src/Powercord/coremods/settings/index.js:51]: Make this array a variable to avoid repetition 
Use `Array.includes` instead of `indexOf`

**Review by asportnoy — CHANGES_REQUESTED** [review]

**Comment by asportnoy** [comment]
Also, the linter needs to be run

**Review by asportnoy — APPROVED** [review]

**Review by Vendicated — APPROVED** [review]

**Pull request merged**